### DISCUSSION TITLE
when I run my celery server,the server cursor is blinking before the ready message(Sloved)

### DISCUSSION THREAD

**Discussion by ranajoy19** [celery/celery#7592] [Q&A]
I'm having a bizarre problem [error](https://drive.google.com/file/d/1UfphwzJ5U3YOa_cFLK66mj5poLFVCY3y/view?usp=sharing) when I run my celery it's loading but not giving the ready message if I run any task it is running the function but not giving any success message with seconds, even I  have make  the function with a print statement to make it come to celery the return inside the function, doesn't  come to celery as it should be normally with the success message with 

this is my celery.py

`from __future__ import absolute_import, unicode_literals
import os
from celery import Celery
os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'ontym.settings')
app = Celery('ontym')
app.config_from_object('django.conf:settings', namespace='CELERY')
app.autodiscover_tasks()`

this is the server my picture [celery server](https://i.stack.imgur.com/b2PlH.png) but it should be like this [Celery server with ready message](https://i.stack.imgur.com/nBsXA.png)

tasks.py

from celery import shared_task
`@shared_task
def hello_task(name):
    print("hello from task",name)`
 
this is my result 
[result](https://i.stack.imgur.com/TzW2T.png)
 
this is inside the __init__.py

` .celery import app as celery
_app __all__=('celery_app',) `

[a comment by an automated account is not shown]

**Reply by ranajoy19** [reply]
It slove , the problem was my logger setup inside the settings file

**Reply by auvipy** [reply]
this was not a bug report rather a coding error. use stackoverflow for this type of issues or create a discussion here for future ref.
